easyApp/Logic/Maintenance.py

```diff
@@ -30,13 +31,9 @@ def __init__(self, parent=None):
         self._silent_check = True
         self._error_message = ""
         self._web_version = ""
-
+        self._web_date = ""
         self._release_notes = ""
-
-        self._process = QProcess()
-        self._process.setWorkingDirectory(QApplication.applicationDirPath())
-        #self._process.setWorkingDirectory("/Applications/easyDiffraction/MaintenanceTool.app/Contents/MacOS")
-        self._process.setProgram(Updater.exeRelativePath())
+        self._process = self._createUpdaterProcess()
```

Comment:
Should the process be initialized here? This means its lifetime is determined by the lifetime of the whole application and its instantiation here leads to longer startup time.
However, keeping track of running instances (making sure you only run it once) and subsequent checks should be noticeably faster, so maybe there is a reason for reusing the instance?